src/composables/allowable-actions-composable.ts

```diff
@@ -0,0 +1,122 @@
+import { computed } from 'vue'
+import { getFeatureFlag } from '@/utils'
+import { AllowableActions, CorpTypeCd, Routes } from '@/enums'
+import { useRoute } from 'vue-router'
+import { useStore } from 'vuex'
+
+const store = useStore()
+
+export const AllowableActionsComposable = () => {
+  const getEntityType = computed(() => store.getters['getEntityType'] as CorpTypeCd)
+  const hasBlocker = computed(() => store.getters['hasBlocker'] as boolean)
+  const isBComp = computed(() => store.getters['isBComp'] as boolean)
+  const isBcCompany = computed(() => store.getters['isBcCompany'] as boolean)
+  const isUlc = computed(() => store.getters['isUlc'] as boolean)
+  const isHistorical = computed(() => store.getters['isHistorical'] as boolean)
+  const isRoleStaff = computed(() => store.getters['isRoleStaff'] as boolean)
+  const isSoleProp = computed(() => store.getters['isSoleProp'] as boolean)
+  const isPartnership = computed(() => store.getters['isPartnership'] as boolean)
+  const isFirm = computed(() => store.getters['isFirm'] as boolean)
+  const isCoop = computed(() => store.getters['isCoop'] as boolean)
+  const hasBlockerExceptStaffApproval = computed(() => store.getters['hasBlockerExceptStaffApproval'] as boolean)
+  const isGoodStanding = computed(() => store.getters[''] as boolean)
```

Comment:
This is really repetitive. Is there a way to combine these and just import/declare a single thing?

Reply:
Or we could write a helper?
Or is there a decorator for this somewhere?

Reply:
Yes this is way to repetitive.  This is what I mean when I think we would need to do a major refactor. As code used anywhere can be reactive this could be replaced with 

const { getEntityType, hasBlcker, ect.... } = getters where all the computed methods are stored